css/stylesheet.css

```
@@ -0,0 +1,47 @@
+header {
+    gap: 12px;
+    display: flex;
+    position: absolute;
+    right: 10px;
+    z-index: 100;
+    top: 10px;
+    margin-top: 15px;
+}
+
+header button {
+    background-color: #022432;
+    padding: 10px 25px;
+    position : relative;
+    margin-right: 15px;
+}
+
+.header-link {
+    color: white;S
```

typo
Unnecessary 's' are included :)
```suggestion
    color: white;
```

---

ご指摘ありがとうございます！
修正しました。

88dcb55